cmd/tailscale/cli/update.go

```diff
@@ -43,7 +55,22 @@ var updateArgs struct {
 	version string // explicit version; empty means auto
 }
 
+// winMSIEnv is the environment variable that, if set, contains makes the update
```

Comment:
"contains makes the update command"
I think "contains" was intended to be removed from the sentence.

Reply:
reworded